src/NHibernate/Tuple/PocoInstantiator.cs

```diff
@@ -103,7 +103,7 @@ private object GetInstance()
 			}
 			if (mappedClass.IsValueType)
 			{
-				return Cfg.Environment.ObjectsFactory.CreateInstance(mappedClass, true);
+				return Activator.CreateInstance(mappedClass, true);
```

Comment:
As discussed in #1758, this usage is not a dependency but an entity instantiation. If the user wants some custom behavior for instantiating value types, he now would have to provide its own tuplizer.

Reply:
I don't think that the `true` here is required.

Reply:
Probably in some partial trust or similar environments.

Reply:
I have just preferred to be quite conservative, inlining the default previous implementation "as is".